gcc/testsuite/rust/compile/match8.rs

```diff
@@ -0,0 +1,29 @@
+enum E {
+    A(),
+    B,
+}
+
+static static_e: E = E::A();
+
+struct S;
```

Comment:
Shouldn't we also add a success test for unit structs?

Reply:
Yes, i also found cases for constants are missing:)